### DISCUSSION TITLE
aiohttp roadmap?

### DISCUSSION THREAD

**Discussion by walkingpendulum** [aio-libs/aiohttp#7789] [Q&A] [answered]
is there somewhere a roadmap? in recent year or so there were only security fixes and various bug fixes. any new features to wait for?

**Comment by Dreamsorcerer** [comment, marked as answer]
Things are merged as and when they are done by volunteers, so there is no roadmap. Read the release notes for 3.9, there are some new things in there, mostly on the server-side.
